pkg/kubebench/kubebench.go

```diff
@@ -86,6 +100,29 @@ func RunJob(params *params.KubeBenchArgs) (*kubebench.OverallControls, error) {
 	return controls, nil
 }
 
+func deploySecret(ctx context.Context, clientset *kubernetes.Clientset, params *params.KubeBenchArgs) (string, error) {
+	// Create a new secret object
+	secret := &apiv1.Secret{
+		Type: apiv1.SecretTypeOpaque,
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "docker-image-pull-secret",
+			Namespace: params.Namespace,
+		},
+		Data: map[string][]byte{
+			".dockerconfigjson": []byte(`{"auths":{"` + params.Registry + `":{"username":"` + params.RegistryUsername + `","password":"` + params.RegistryPassword + `"}}}`),
```

Comment:
If registry param is specified, but not the registryUsername and/or registryPassword, we would still create the secret with null value in the json value?

Reply:
resolved this @lalalawoo